test/e2e/next-test-lib/next-test-utils.js

```diff
@@ -128,7 +128,9 @@ export function renderViaHTTP(appPort, pathname, query, opts) {
 export async function fetchViaHTTP(appPort, pathname, query = undefined, opts = undefined, useUndici = false) {
   const url = `${pathname}${typeof query === 'string' ? query : query ? `?${qs.stringify(query)}` : ''}`
   const fetch = useUndici ? undiciFetch : nodeFetch
-  return fetch(getFullUrl(appPort, url), opts)
+  const fullUrl = getFullUrl(appPort, url)
+  console.log(`Fetching url: ${fullUrl}`)
```

Comment:
Was this for debugging purposes only?

Reply:
No, I've added them to the test script because it makes it a lot easier to debug failures if you can see the actual URL that was loaded